.storybook/main.js

```diff
@@ -21,6 +22,23 @@ module.exports = {
     '@storybook/addon-storysource',
   ],
   webpackFinal: async config => {
+    // Get the specifications object and replace with a real object in the spec.ts file
+    const specs = await getSpecifications();
+    // modules/specifications/lib/specs.ts
+    config.module.rules.push({
+      test: /specs\.ts$/,
+      include: [path.resolve(__dirname, '../modules/specifications')],
+      use: [
+        {
+          loader: require.resolve('string-replace-loader'),
+          options: {
+            search: '[/* SPEC_FILES_REPLACE_BY_WEBPACK */]',
+            replace: JSON.stringify(specs, null, '  '),
+          },
+        },
+      ],
+    });
+
```

Comment:
This is how specifications in docs works in Webpack. There is a separate process for build to publish specifications.